include/deal.II/base/mpi.h

```diff
@@ -238,6 +240,19 @@ namespace Utilities
     create_ascending_partitioning(const MPI_Comm &           comm,
                                   const IndexSet::size_type &local_size);
 
+#ifdef DEAL_II_WITH_MPI
+    /**
+     * Calculate mean and standard deviation across the MPI communicator @p comm
+     * for values provided as a range `[begin,end)`.
+     *
+     * @note The standard deviation is calculated using the `N - 1` divisor,
+     * where `N` is the total size across the communicator.
```

Comment:
Could you just write a formula? I suspect that you want to say something like "The mean is computed as $\bar x=\frac 1N \sum x_k$ where the $x_k$ are the elements pointed to by the begin and end iterators on all processors (i.e., each processor's [b,e) range points to a subset of the overall number of elements)." Then also show the formula for the standard deviation.

I think that if you divide by N-1 instead of N in the computation, then this is commonly called the "sample variance", see https://en.wikipedia.org/wiki/Standard_deviation#Uncorrected_sample_standard_deviation